pkg/expr/query.types.json

```diff
@@ -96,10 +96,12 @@
               "description": "Reducer Options",
               "properties": {
                 "mode": {
-                  "description": "Non-number reduce behavior\n\n\nPossible enum values:\n - `\"dropNN\"` Drop non-numbers\n - `\"replaceNN\"` Replace non-numbers",
+                  "description": "Non-number reduce behavior\n\n\nPossible enum values:\n - `\"dropNN\"` Drop non-numbers\n - `\"replaceNN\"` Replace non-numbers\n - `\"__expr__\"` \n - `\"-100\"` ",
                   "enum": [
                     "dropNN",
-                    "replaceNN"
+                    "replaceNN",
+                    "__expr__",
+                    "-100"
```

Comment:
hymm -- these do not look like valid properties.

If you revert and try again do you get the same thing?

Reply:
Let me try

Reply:
At first got the same result. Then I ran `go mod tidy` and after that no longer 🤷 

It did happen in CI here very strange https://drone.grafana.net/grafana/grafana-enterprise/68816/2/7